### ISSUE TITLE
Add support for TeamCity Build Script Interaction tokens

### ISSUE THREAD

**Issue by anurse** [pester/Pester#242]
More info: https://confluence.jetbrains.com/display/TCD9/Build+Script+Interaction+with+TeamCity

Basically, when running Pester tests as part of a TeamCity build, it would be nice to be able to get the pretty Test list showing the status of all the Pester tests that were run as part of the build, for example:

![image](https://cloud.githubusercontent.com/assets/7574/5493077/645e62ae-869c-11e4-9e41-a699c2470447.png)

I've got a PR that will be coming right away that adds this feature, so if you're up for it, there isn't even much work to be done! :)

We're going to be using Pester to test the 'kvm.ps1' script that is part of ASP.NET 5 (see: https://github.com/aspnet/kvm/tree/anurse/kvm-test/test/ps1) and we'd love to be able to get rid of the tiny fork of Pester I made to add this feature :P.

**Comment by matt-richardson** [comment]
I've found another use case where this kind of reporting would be better than using the "xml report processing" feature: the processing is async, so the build log is out of order.

We are iterating over a bunch of files, and invoking pester on each one. We're using teamcity build messages to group the output for each one, but as the processing is async, the reporting of errors does not stay within the nested group - they bleed over into the middle of the next pester run.

I've tried the [alternative method](https://confluence.jetbrains.com/display/TCD9/Build+Script+Interaction+with+TeamCity#BuildScriptInteractionwithTeamCity-ImportingXMLReports) of reporting build results, but no luck.
